### DISCUSSION TITLE
Future Lazer Skin Customizability

### DISCUSSION THREAD

**Discussion by eff3ry** [ppy/osu#23278] [Ideas] [answered]
To my knowledge the current default lazer skins are generated with code rather than assets,
from this I think it wouldn't be unreasonable to assume custom skins could be made in the same way in the future, using some system of scripting to generate skins etc.
If or when such a feature is implemented we could have custom variables defined in the skin file that appear under the skin selector for in-game customization i.e. border-thickness, colour, transparency etc.
Let me know your thoughts, and thank you for reading!

**Comment by peppy** [comment, marked as answer]
yes, that is eventually the plan.

**Reply by eff3ry** [reply]
haha thanks 😅
